libraries/AP_AHRS/AP_AHRS_Backend.h

```diff
@@ -76,12 +76,20 @@ class AP_AHRS_Backend
 
     // get the index of the current primary accelerometer sensor
     virtual uint8_t get_primary_accel_index(void) const {
+#if AP_INERTIALSENSOR_ENABLED
```

Comment:
under what circumstances can we have AHRS without INS?

Reply:
ExternalAHRS is one example.  Probably the only example by definition.

But I made these change because we're in the header file, and didn't wish to consider the implications of moving this implementation into the .cpp file in terms of flash and performance.